modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,260 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// (link:https://issues.redhat.com/browse/SRVKP-2309[SRVKP-2309 Pipeline as code: validate repository URL uniqueness])
+// Puneet Punamiya
+
+* With this update, as a cluster administrator, if you enable your local {tekton-hub} instance, it periodically refreshes the database so that changes in the catalog appear in the {tekton-hub} web console. You can adjust the period between refreshes.
++
+Previously, to add the tasks and pipelines in the catalog to the database, you performed that task manually or set up a cron job to do it for you.
+// (link:https://issues.redhat.com/browse/SRVKP-2252[SRVKP-2252 Refresh catalogs on schedule])
+// Puneet Punamiya
+
+* With this update, you can install and run a {tekton-hub} instance with minimal configuration. This way, you can start working with your teams to decide which additional customizations they might want.
+// (link:https://issues.redhat.com/browse/SRVKP-2250[SRVKP-2250 Install Hub with default configs])
+// Shiv Verma
+
+// [Chains]
+* Previously, the `chains-config` config map for {tekton-chains} in the `openshift-pipelines` namespace was automatically reset to default after upgrading the {pipelines-title} Operator. The current update fixes this issue. Now, you can configure key-value pairs in the `chains-config` config map, and those values persist after upgrading the {pipelines-title} Operator.
+// https://issues.redhat.com/browse/SRVKP-2349
+// Piyush Garg @piyush-garg
+
+* This update adds `GIT_SSL_CAINFO` to the `git-clone` task so you can clone secured repositories.
+// https://issues.redhat.com/browse/SRVKP-1423
+// @savitaashture
+
+[id="hub-new-features-1-8_{context}"]
+=== Hub
+
+* With this update, you can disable the login and rating features of your {tekton-hub} web console. To do so, omit adding a value for the `AUTH_BASE_URL` field in the `<tekton_hub_repository>/config/02-api/20-api-secret.yaml` file.
+// https://github.com/tektoncd/hub/pull/542
+// Shiv Pratap Verma @pratap0007
+
+* With this update, you don't need to log in to the {tekton-hub} web console to add resources from the catalog to the database. Now, these resources are automatically added when the catalog refresh API runs.
+// https://github.com/tektoncd/hub/pull/510
+// Shiv Pratap Verma @pratap0007
+
+* This update automatically refreshes the catalog every 30 minutes by calling the catalog refresh API job. This interval is user-configurable.
+// https://github.com/tektoncd/hub/pull/547
+// Shiv Pratap Verma @pratap0007
+
+[id="pac-new-features-1-8_{context}"]
+=== {pac}
+// {pac} owners: Shivam Mukhade Savita Ashture
+
+:FeatureName: {pac} (PAC)
+include::snippets/technology-preview.adoc[]
+
+* {pipelines-title} 1.7.x shipped with {pac} 0.5.x. The current update ships with {pac} 0.10.x. This change creates a new route in the `openshift-pipelines` namespace for the new controller. You must update this route in GitHub Apps or webhooks that use {pac}. To fetch the route, use the following command:
```

Comment:
```suggestion
* {pipelines-title} 1.7.x shipped with {pac} 0.5.x. The current update ships with {pac} 0.10.x. This change creates a new route in the `openshift-pipelines` namespace for the new controller. You must update this route in GitHub Apps or webhooks that use {pac}. 

To fetch the route, use the following command:
```

Should we add a line before saying `To fetch the route, use the following command:` :thinking:

Reply:
It's not necessary to do so. In release notes, we prefer a more compact format.